cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/SourceControlManagementHttpHandlerTests.java

```diff
@@ -140,6 +160,23 @@ public void testInvalidRepoConfig() throws Exception  {
     assertResponseCode(200, response);
   }
 
+  @Test
+  public void testFeatureFlagDisabledEndpoint() throws Exception {
+    setSCMFeatureFlag(false);
+
+    // Set repository config
+    RepositoryConfig namespaceRepo = new RepositoryConfig.Builder().setProvider(Provider.GITHUB)
+      .setLink(LINK).setDefaultBranch(DEFAULT_BRANCH).setAuthType(AuthType.PAT)
+      .setTokenName(TOKEN_NAME).setUsername(USERNAME).build();
+
+    // Assert the namespace does not exist
+    assertResponseCode(403, setRepository(NAME, createRepoRequestString(namespaceRepo)));
+    assertResponseCode(403, getRepository(NAME));
+    assertResponseCode(403, deleteRepository(NAME));
+
+    setSCMFeatureFlag(true);
```

Comment:
better to set this in an after/ clean up class method. What if this test fails? then we might not reach this line

Reply:
nice catch! will add it

Reply:
actually we just need to remove this line, since I have `setSCMFeatureFlag(true);` in `@BeforeClass`